Sources/Hyperspace/Service/Backend/BackendService.swift

```diff
@@ -31,6 +32,16 @@ public class BackendService {
 // MARK: - BackendService Conformance to BackendServiceProtocol
 
 extension BackendService: BackendServiceProtocol {
```

Comment:
In addition, this can be an extension to `BackendServiceProtocol` as it just calls existing requirements in the `BackendServiceProtocol`. That way it doesn't need to be implemented. Whether we want to list it as a protocol requirement so it can be customized is another story.

Reply:
I moved this into an extension on `BackendServiceProtocol`. I left the function as part of the requirements for now because I don't see a reason not to, but let me know if that's an issue.